tds.go

```diff
@@ -965,6 +966,27 @@ func parseConnectParams(dsn string) (connectParams, error) {
 		}
 	}
 
+	// https://msdn.microsoft.com/en-us/library/dd304019.aspx
```

Comment:
Removed, could not find the exact HTML version of the page within the PDF specification. As that provides the protocol limits